server/paymentProviders/stripe/creditcard.js

```diff
@@ -231,7 +234,18 @@ const createChargeAndTransactions = async (hostStripeAccount, { order, hostStrip
 };
 
 /**
- * Given a charge id, retrieves its correspind charge and refund data.
+ * Handles the zero-decimal currencies for Stripe; https://stripe.com/docs/currencies#zero-decimal
+ */
+const zeroDecimalCurrencyValue = (currency, amount) => {
```

Comment:
It's not a good descriptive name for what it's doing.

What does it do? It converts an amount to Stripe's expected format. What is the most important thing? The amount?

I would then propose, `convertToStripeAmount = (amount, currency) =>`

You will later need the opposite, `convertFromStripeAmount = (amount, currency) =>`

Reply:
Thanks. I've renamed the functions. 👍🏽